### PR TITLE
Dlgstackdesign

### LINKED ISSUE #5831: Stack (melt) dialog design
This dialog has minor designer issues see below-highlighted screenshots.

image

image

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/developers this is ready to review

Relates to #5831 but a new aspect has been noted in #5831 after this pull request is merged. 

Changes:
* Design changes to "tidy up" the stack dialog

Unable to fix the ucrSave problem - this seems to be an issue on similar dialogs (aside from when the "Position" button is visible?). Is there a way to do this that I am missing, and if not has this been recorded as an issue?

Before:
image

After:
image

image

### CODE REVIEW

**Comment by shadrackkibet** [comment]
Looks good to me. @rdstern @lloyddewit review and approve for merging.

File: instat/dlgStack.vb [hunk 1]
```diff
@@ -82,10 +82,10 @@ Public Class dlgStack
         ucrNudNoSets.Minimum = 2
         ucrNudNoSets.SetLinkedDisplayControl(lblSets)
 
-        ucrSaveNewDataName.SetIsTextBox()
-        ucrSaveNewDataName.SetLabelText("New Data Frame Name:")
         ucrSaveNewDataName.SetSaveTypeAsDataFrame()
         ucrSaveNewDataName.SetDataFrameSelector(ucrSelectorStack.ucrAvailableDataFrames)
+        ucrSaveNewDataName.SetLabelText("New Data Frame Name:")
+        ucrSaveNewDataName.SetIsTextBox()
```
Comment (shadrackkibet) [inline instat/dlgStack.vb:88]: Apart from making code more readable and in logical order. Is there a functionality change by changing the order?

Comment (lilyclements) [reply instat/dlgStack.vb:88]: Yes, see the issue in #5866

Comment (shadrackkibet) [reply instat/dlgStack.vb:88]: That is surprising to me I didn't expect it. Thanks

**Comment by shadrackkibet** [comment]
@rdstern @lloyddewit this is ready for you to review and approve for merging.

**Comment by lloyddewit** [comment]
@rdstern Please could you test before I review? thanks

**Pull request merged**